# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: config/system-headers.mozbuild
# Commit: 9973c75413d6
# Full Hash: 9973c75413d6237b9005b9ab908898a153bd2f06
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 7c947114da80 -r 9973c75413d6 config/system-headers.mozbuild
--- a/config/system-headers.mozbuild	Tue Feb 27 12:40:22 2024 +0000
+++ b/config/system-headers.mozbuild	Tue Feb 27 12:40:22 2024 +0000
@@ -1267,11 +1267,6 @@
         "jpeglib.h",
     ]
 
-if CONFIG["MOZ_LIBAV_FFT"]:
-    system_headers += [
-        "libavcodec/avfft.h",
-    ]
-
 if CONFIG["MOZ_SYSTEM_PNG"]:
     system_headers += [
         "png.h",